src/test/scala/is/hail/methods/AggregatorSuite.scala

```diff
@@ -20,7 +20,7 @@ class AggregatorSuite extends SparkSuite {
       .annotateVariantsExpr(
         """va.test.callrate = gs.fraction(g => g.isCalled()),
           |va.test.AC = gs.map(g => g.nNonRefAlleles()).sum(),
-          |va.test.AF = gs.map(g => g.nNonRefAlleles()).stats().sum.toDouble() / gs.filter(g => g.isCalled()).count() / 2.0,
+          |va.test.AF = gs.map(g => g.nNonRefAlleles()).stats().sum.toFloat() / gs.filter(g => g.isCalled()).count() / 2.0,
```

Comment:
I'll remove the aliases for now. If we want to add them back in before 0.2, we can always do that.